cpp/include/copc-lib/las/laz_config.hpp

```diff
@@ -0,0 +1,59 @@
+#ifndef COPCLIB_LAZ_CONFIG_H_
+#define COPCLIB_LAZ_CONFIG_H_
+
+#include <memory>
+#include <string>
+
+#include "copc-lib/las/header.hpp"
+#include "copc-lib/las/utils.hpp"
+#include "copc-lib/las/vlr.hpp"
+
+namespace copc
+{
+class CopcConfig;
+
+namespace las
+{
+const int LAZ_HEADER_SIZE = 375;
+
+class LazConfig
+{
+  public:
+    LazConfig() = default;
+    LazConfig(const las::LasHeader &header, const std::string &wkt, const las::EbVlr &extra_bytes_vlr)
+        : header_(std::make_shared<las::LasHeader>(header)), wkt_(wkt),
```

Comment:
let's move the `make_shared` into the constructor itself, so there's not a bunch of logic in these initializers